network_observability/network-observability-overview.adoc

```diff
@@ -37,7 +37,41 @@ In *Observe* -> *Dashboards*, the *Netobserv* dashboard provides a quick overvie
  * *Top byte rates received per source and destination workloads*
 
 *Infrastructure* and *Application* metrics are shown in a split-view for namespace and workloads.
-You can configure the `FlowCollector` `spec.processor.metrics` to add or remove metrics by changing the `ignoreTags` list. For more information about available tags, see the xref:../network_observability/flowcollector-api.adoc#network-observability-flowcollector-api-specifications_network_observability[Flow Collector API Reference]
+You can configure the `FlowCollector` custom resource `spec.processor.metrics.includeList` to add or remove metrics.
+
+Available metrics names are as follows (names followed by * are enabled by default): 
+
+* namespace_egress_bytes_total
+* namespace_egress_packets_total
+* namespace_ingress_bytes_total
+* namespace_ingress_packets_total
+* namespace_flows_total *
+* node_egress_bytes_total
+* node_egress_packets_total
+* node_ingress_bytes_total *
+* node_ingress_packets_total
+* node_flows_total
+* workload_egress_bytes_total
+* workload_egress_packets_total
+* workload_ingress_bytes_total *
+* workload_ingress_packets_total
+* workload_flows_total
+
+When the `PacketDrop` feature is enabled in `spec.agent.ebpf.features` (with `privileged` mode), additional metrics are available:
```

Comment:
```suggestion
When the `PacketDrop` feature is enabled in `spec.agent.ebpf.features` (with `privileged` mode), the following additional metrics are available:
```

Reply:
I'll incorporate this in this PR: https://github.com/openshift/openshift-docs/pull/69106/files